aws/terraform/metaflow/example.tfvars

```diff
@@ -0,0 +1,21 @@
+env = "prod"
+
+aws_region = "us-west-2"
+
+# Since we're testing, as this is the dev stack setting the min and desired to 0 so that we experience a cold-boot
```

Comment:
Maybe reword it - `Setting min vpcus and desired vcpus to 0 to prevent accidental cost accumulation. These settings will result in elongated job startup times as AWS boots up the necessary compute resources.`